### PR TITLE
feat: add cv::wechat_qrcode::WeChatQRCode

### PR DESCRIPTION
**Pull request by dirname**
fix: https://github.com/hybridgroup/gocv/pull/918#issuecomment-1062658077

@deadprogram

### CODE REVIEW

**Review by dirname — COMMENTED** [review]
review: ready to merge

File: wechat_qrcode_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package gocv
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestNewWeChatQRCode(t *testing.T) {
```
Comment (deadprogram) [inline wechat_qrcode_test.go:9]: Hello @dirname 

I am thinking that probably the tests and test model files should be handled conditionally like the test files for Caffe/Tensorflow. 

Please see https://github.com/hybridgroup/gocv/blob/release/dnn_test.go#L98

The reason for this is to keep the size of the repo down to a smaller size. 

Is there a place that the model files you are using for the tests here can be downloaded from, similar to how it is handled here for CircleCI: https://github.com/hybridgroup/gocv/blob/release/.circleci/config.yml#L25-L30 

and then here for Appveyor: https://github.com/hybridgroup/gocv/blob/release/appveyor_build_opencv.cmd#L19

Comment (dirname) [reply wechat_qrcode_test.go:9]: @deadprogram Changed, please review again.

**Review by dirname — COMMENTED** [review]
fix: https://github.com/hybridgroup/gocv/pull/949#discussion_r825261378

File: wechat_qrcode_test.go (hunk 1 above)
Comment (deadprogram) [reply wechat_qrcode_test.go:9]: Thanks for the quick update!

**Comment by deadprogram** [comment]
@dirname do you mind squashing these commits together, or would you prefer that I do it before merge?

**Comment by dirname** [comment]
@deadprogram You can choose the way you like, whether there is an estimated release time for the next release version, I want to be able to use the new features in the release version as soon as possible.

**Comment by deadprogram** [comment]
I can squash and merge right now.

The next release is about to go out next week, so good timing!

**Comment by dirname** [comment]
> I can squash and merge right now.
> 
> The next release is about to go out next week, so good timing!

Thanks for your awesome work, I will contribute as much as I can.

**Comment by deadprogram** [comment]
I think I perhaps merged too soon. Just realized that this is part of `opencv_contrib` so really it needs to get moved to the `contrib` package inside of GoCV.

If you have time perhaps you can look at that please? Otherwise I will look into it this weekend.

**Comment by dirname** [comment]
> I think I perhaps merged too soon. Just realized that this is part of `opencv_contrib` so really it needs to get moved to the `contrib` package inside of GoCV.
> 
> If you have time perhaps you can look at that please? Otherwise I will look into it this weekend.

https://github.com/hybridgroup/gocv/pull/952

**Pull request merged**